tasks/test.rake

```diff
@@ -17,6 +17,12 @@ namespace 'test' do
     t.cucumber_opts = 'features --format progress --color'
   end
 
+  task 'enable_simplecov' do
+    ENV['SIMPLECOV'] = '1'
+  end
+
+  task 'spec' => 'test:enable_simplecov'
```

Comment:
If you enable it on every spec run, why not drop that ENV charade completely and.....just require it unconditionally on spec run?